### PR TITLE
Audio: EQ IIR: Add support for large configuration blobs plus EQ topology fixes

### PR DESCRIPTION
**Pull request by singalsu**
The main change in this pull request is to handle multi-part configuration IPC. The other changes are align of bytes control max. size in topologies to 1024 bytes to IIR and 4096 bytes to FIR. Also a mistake fix for macro name clear is included. It didn't seem cause any issues though.

### CODE REVIEW

**Comment by singalsu** [comment]
SOFCI TEST

File: src/audio/eq_iir/eq_iir.c [hunk 1]
```diff
@@ -652,21 +657,31 @@ static int iir_cmd_get_data(struct comp_dev *dev,
...
-				  bs);
-			if (bs > SOF_EQ_IIR_MAX_SIZE || bs == 0 ||
-			    bs > max_size)
-				return -EINVAL;
-			ret = memcpy_s(cdata->data->data,
-				       ((struct sof_abi_hdr *)
-				       (cdata->data))->size, cd->config, bs);
+			cdata->elems_remaining = 0;
+			offset = 0;
+			if (bs > max_size) {
```
Comment (lgirdwood) [inline src/audio/eq_iir/eq_iir.c:665]: what are we doing here ? maybe some comments needed.

Comment (singalsu) [reply src/audio/eq_iir/eq_iir.c:665]: Good that you asked! I copied this from FIR and it seems there's more change need (both components). I seems that the get_data() for multi-part happens only for a topology embedded long blob and I didn't test it, I tested before this only with sof-ctl. There seems to be a similar memcpy_s() assert related panic happening that I need to fix. The set_data() worked OK as far as I see prior to crash.

Comment (singalsu) [reply src/audio/eq_iir/eq_iir.c:665]: I think I found the mistake with size. I'll update this PR tomorrow.

Comment (singalsu) [reply src/audio/eq_iir/eq_iir.c:673]: I had from above the headers size subtraction missing from max_size. It caused the assert() in previous PR version. Now both topology embedding and sof-ctl ways for setup work. I've added comments to code to understand what happens.

**Comment by lgirdwood** [comment]
CI usual issues.

**Pull request merged**